runtimes/alibi-explain/mlserver_alibi_explain/explainers/black_box_runtime.py

```diff
@@ -38,16 +38,15 @@ def __init__(self, settings: ModelSettings, explainer_class: Type[Explainer]):
         super().__init__(settings, explainer_settings)
 
     async def load(self) -> bool:
-        # get the metadata of the underlying inference model via v2 metadata endpoint
-        self.infer_metadata = remote_metadata(construct_metadata_url(self.infer_uri))
+        self.infer_metadata = None
 
         # TODO: use init explainer field instead?
         if self.alibi_explain_settings.init_parameters is not None:
             init_parameters = self.alibi_explain_settings.init_parameters
             init_parameters["predictor"] = self._infer_impl
             self._model = self._explainer_class(**init_parameters)  # type: ignore
         else:
```

Comment:
should we make ` self._model = self._explainer_class(**init_parameters) ` async as well? perhaps add an issue to consider.

Reply:
we probably need to look more into the behaviour of async. but in the case of
-  `_load_from_uri`, which will call `alibi.load_explainer` 
- and in the case of `_explain_impl`, which will call `alibi.Explainer.explain`

These calls are not `async` in alibi but we still managed to get them to run asynchronously in mlserver. Given that we dont have an immediate use case for `_explainer_class` it is probably going to require more testing, happy for it to be done as a separate ticket if you feel so. So perhaps add an issue so that we dont forget.

Reply:
yes I suppose that would also need to be added to an async event loop call. Maybe need more example tests in MLServer code for this so maybe a future issue/PR?